File: swagger-client/dart/lib/model/send_mail_to.dart
part of swagger.api;

class SendMailTo {
  
  SendMailTo();

  @override
  String toString() {
    return 'SendMailTo[]';
  }

  SendMailTo.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
  }

  Map<String, dynamic> toJson() {
    return {
     };
  }

  static List<SendMailTo> listFromJson(List<dynamic> json) {
    return json == null ? new List<SendMailTo>() : json.map((value) => new SendMailTo.fromJson(value)).toList();
  }

  static Map<String, SendMailTo> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, SendMailTo>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new SendMailTo.fromJson(value));
    }
    return map;
  }
}

File: openapi-client/cpp-tiny/lib/Models/SendMailTo.cpp


#include "SendMail_to.h"

using namespace Tiny;

SendMail_to::SendMail_to()
{
}

SendMail_to::SendMail_to(std::string jsonString)
{
	this->fromJson(jsonString);
}

SendMail_to::~SendMail_to()
{

}

void
SendMail_to::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);


}

bourne::json
SendMail_to::toJson()
{
    bourne::json object = bourne::json::object();


    return object;

}



